imap_processing/ultra/l0/decom_ultra.py

```diff
@@ -257,7 +261,9 @@ def process_ultra_rates(sorted_packets: list, decom_data: dict):
             ULTRA_RATES.mantissa_bit_length,
         )
 
-        for index in range(ULTRA_RATES.len_array):
+        for index in range(ULTRA_RATES.len_array):  # type: ignore[arg-type]
+            # TODO Change, Argument 1 to "range" has incompatible type
+            # "Optional[int]"; expected "SupportsIndex"
```

Comment:
```suggestion
        for index in range(ULTRA_RATES.len_array):  # type: ignore[arg-type]
```

Another weird corner case that I think can be ignored